### ISSUE TITLE
Mismatch of uppercase / lowercase in java.org.xbib.elasticsearch.action.isbnformat

### ISSUE THREAD

**Issue by blured75** [jprante/elasticsearch-plugin-bundle#18]
Hi,

For information, there is some problem in source code which does not compile in master branch (perhaps it is normal) : 
the files IsbnFormatRequest.java & IsbnFormatResponse.java should be named ISBN.... based on the classnames.

Regards,
Blured.

**Comment by jprante** [comment]
Yes, I already noticed. It's Mac OSX file system annoyance. Will be fixed asap.

**Comment by blured75** [comment]
Thanks
